src/Util/Test.php

```diff
@@ -353,9 +353,48 @@ public static function getDependencies(string $className, string $methodName): a
             );
         }
 
+        // Normalize dependency name to className::methodName
```

Comment:
Perhaps it makes sense to extract a method here, so we can replace the comment with a suitable method name?

Reply:
I agree and this will surely happen in the near future, as I'll keep cleaning up parts of the runner and support logic while working on the Iterator and data providers.

The word 'dependency' is used in multiple contexts at the moment:
- the parsed user input directly from `@depends`
- the dependencies as expressed in the internal test collection tree
- as a bonus it causes the return value of a dependency to be inserted as parameter